src/main/java/com/jfb/tickets/dao/TicketDAO.java

```diff
@@ -0,0 +1,121 @@
+package com.jfb.tickets.dao;
+
+import com.jfb.tickets.DatabaseConnection;
+import com.jfb.tickets.model.Ticket;
+
+import java.util.UUID;
+import java.util.List;
+import java.util.ArrayList;
+
+import java.time.Instant;
+
+import java.sql.SQLException;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.Timestamp;
+import java.sql.Connection;
+
+public class TicketDAO {
+    private final DatabaseConnection databaseConnection;
+
+    public TicketDAO() {
+        databaseConnection = new DatabaseConnection();
+    }
+
+    public Ticket get(UUID id) {
+        String sql = "SELECT * FROM tickets WHERE id = ?";
```

Comment:
perfectly will be to make query strings as static variables related to the class (optional)